man/ocaml.1

```diff
@@ -190,6 +195,21 @@ interactive session.
 .B \-strict\-sequence
 Force the left-hand part of each sequence to have type unit.
 .TP
+.B \-no-strict\-sequence
+Left-hand part of a sequence need not have type unit.
+This is the default.
+.TP
+.B \-strict\-formats
+Reject invalid formats that were accepted in legacy format
+implementations. You should use this flag to detect and fix
+such invalid formats, as they will be rejected by future
+OCaml versions.
```

Comment:
I would rather have this change in a separate PR because the text in the development branch and in the 5.0 branch needs to be different.

Reply:
Would you like me to remove the changes made in the ocaml.1 file and change back it to the default?

Reply:
I was speaking of the `-strict-format` and `-no-strict-format` change rather than the changes in `ocaml.1`.

Reply:
Oh alright. So I'll need to take out the -strict-format and -no-strict-format changes made in all three files for now and later create a separate PR to add the changes, right?

Reply:
Indeed. Moreover you would need to have one version of the PR for the `5.0` branch and another for the `trunk` branch because `-strict-format` is now the default on the `trunk` branch.